src/fromager/overrides.py

```diff
@@ -83,6 +83,17 @@ def patches_for_source_dir(
     return sorted((patches_dir / source_dir_name).glob("*.patch"))
 
 
+def patches_for_req(patches_dir: pathlib.Path, dir_name: str) -> list[pathlib.Path]:
+    """
+    This function will return patches for a specific requirement.
+    It takes in patches directory and requirement as input.
```

Comment:
Okay, changed it to `get_patches()`
Is it more generic? Should I change it?

Reply:
The docstring is out of date. What is `dir_name` expected to be?

This function is returning all patches for all versions of a dependency, starting from a source directory, right? If that's the case, please change `dir_name` to `source_root_dir` to match the value used elsewhere. Naming it consistently helps the caller understand what it is.

Since the function doesn't return patch filenames, it shouldn't be called `get_patches`. Maybe `get_patch_directories`?